### PR TITLE
NguyenH Tien

### PR DESCRIPTION
**Pull request by nxhftieens**
I did something here with the given data. Please check
- raw data is reformatted and divided into 2 separated files: dataset1 and dataset2 for creating 2 different models (I've just kind of finished on the first one)
- the input data set (dataset1) is cleaned, reformatted and split into different sets: train, validation, test. Data is also standardized in the hope of increasing model performance
- in training stage model is created and train with the train dataset, then with models is tuned using a GridSearchCV to find best hyperparameters (alpha, l1_ratio)
- in validation stage different models are evaluated with validation dataset to find which provides best performance
- in test stage the best model in validation is evaluated 1 more time with test dataset
- in production stage a new model is build up with the full dataset.
While working with the models, i have some problems with models' performance.
And there are still some issue with dvc when i tried to use $ dvc repro -s with some stages.
If in terminal i run the command in cmd block of each stage, everything works. But issues occur if i use $ dvc repro -s

### CODE REVIEW

File: .idea/LR.iml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (LlirikOknessu) [inline .idea/LR.iml]: .idea убрать из коммита

**Comment by LlirikOknessu** [comment]
Still, I achieved good accuracy. See, it's really about normalization. You forget to normalize the output variable. In your code you just normalize all variables in X dataset and forget about y, because of that linear regression inadequately fits coefficients. Please do this and don't use ElasticNet with the parameters you have. They don't fit at all.
image

**Comment by LlirikOknessu** [comment]
I just add this line to your code and it works!)
image

**Review by LlirikOknessu — CHANGES_REQUESTED** [review]
See the other comments :)

**Comment by nxhftieens** [comment]
Thank you, I'll try again with normalization on Y datasets soon, at the moment still pretty busy with other subjects)

File: requirementsVSCode.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+matplotlib
```
Comment (LlirikOknessu) [inline requirementsVSCode.txt:1]: Remove this file in the next pull request, please)

File: setupVSCode.txt [hunk 3]
```diff
@@ -0,0 +1,28 @@
+1. Clone repository
```
Comment (LlirikOknessu) [inline setupVSCode.txt:1]: Remove this file in the next pull request, please)

**Review by LlirikOknessu — APPROVED** [review]

**Pull request merged**